docs/Lua API.rst

```diff
@@ -1188,6 +1188,244 @@ Job module
 Units module
 ------------
 
+* ``dfhack.units.isUnitInBox(unit,x1,y1,z1,x2,y2,z2)``
+
+  The unit is within the specified coordinates.
+
+* ``dfhack.units.isActive(unit)``
+
+  The unit is active (alive and on the map).
+
+* ``dfhack.units.isVisible(unit)``
+
+  The unit is visible on the map.
+
+* ``dfhack.units.isCitizen(unit[,ignore_sanity])``
+
+  The unit is an alive sane citizen of the fortress; wraps the
+  same checks the game uses to decide game-over by extinction,
+  with an additional sanity check. You can identify citizens,
+  regardless of their sanity, by passing ``true`` as the optional
+  second parameter.
+
+* ``dfhack.units.isFortControlled(unit)``
+
+  Similar to ``dfhack.units.isCitizen(unit)``, but is based on checks
+  for units hidden in ambush, and includes tame animals. Returns *false*
+  if not in fort mode.
+
+* ``dfhack.units.isOwnCiv(unit)``
+
+  The unit belongs to the player's civilization.
+
+* ``dfhack.units.isOwnGroup(unit)``
+
+  The unit belongs to the player's group.
+
+* ``dfhack.units.isOwnRace(unit)``
+
+  The unit belongs to the player's race.
+
+* ``dfhack.units.isAlive(unit)``
+
+  The unit isn't dead or undead.
+
+* ``dfhack.units.isDead(unit)``
+
+  The unit is completely dead and passive, or a ghost. Equivalent to
+  ``dfhack.units.isKilled(unit) or dfhack.units.isGhost(unit)``.
+
+* ``dfhack.units.isKilled(unit)``
+
+  The unit has been killed.
+
+* ``dfhack.units.isSane(unit)``
+
+  The unit is capable of rational action, i.e. not dead, insane, zombie, or active werewolf.
+
+* ``dfhack.units.isCrazed``
+
+  The unit is berserk and will attack all other creatures except members of its own species
+  that are also crazed. (can be modified by curses)
+
+* ``dfhack.units.isGhost(unit)``
+
+  The unit is a ghost.
+
+* ``dfhack.units.isHidden(unit)``
+
+  The unit is hidden to the player, accounting for sneaking. Works for any game mode.
+
+* ``dfhack.units.isHidingCurse(unit)``
+
+  The unit is hiding a curse.
+
+
+* ``dfhack.units.isMale(unit)``
+
+  The unit is male.
+
+* ``dfhack.units.isFemale(unit)``
+
+  The unit is female.
+
+* ``dfhack.units.isBaby(unit)``
+
+  The unit is a baby.
+
+* ``dfhack.units.isChild(unit)``
+
+  The unit is a child.
+
+* ``dfhack.units.isAdult(unit)``
+
+  The unit is an adult.
+
+* ``dfhack.units.isGay(unit)``
+
+  The unit is gay.
+
+* ``dfhack.units.isNaked(unit)``
+
+  The unit is naked.
+
+* ``dfhack.units.isVisiting(unit)``
+
+  The unit is visiting. eg. Merchants, Diplomatics, travelers.
+
+
```

Comment:
```suggestion

```

Reply:
unless these extra lines (here an elsewhere in this PR) are here for readability, I'd say get rid of them to keep the document consistent

Reply:
They are there for readability, they can parsed out. But they match the header's spacing